package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "telegram-bots",
-  "version": "4.15.0",
+  "version": "4.13.0",
```

Comment:
Please remove the duplicate `version` key

Reply:
Whoops sorry, will be corrected immediately :o